packages/bot-web-ui/src/pages/server-side-bot/performance-panel/performance-panel.tsx

```diff
@@ -3,18 +3,30 @@ import { Tabs } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import Journal from '../journal';
 import Summary from '../summary';
+import ClearJournalTransactions from '../bot-list/clear-journal-transactions';
 
-const PerformancePanel: React.FC = () => {
+const PerformancePanel = () => {
     const [active_index, setActiveTabIndex] = React.useState(0);
+    const [is_clear_dialog_visible, setClearDialogVisibility] = React.useState(false);
     return (
-        <Tabs active_index={active_index} onTabItemClick={setActiveTabIndex} top className='ssb'>
-            <div id='db-run-panel-tab__summary' label={localize('Summary')}>
-                <Summary />
-            </div>
-            <div id='db-run-panel-tab__journal' label={localize('Journal')}>
-                <Journal setActiveTabIndex={setActiveTabIndex} />
-            </div>
-        </Tabs>
+        <>
+            <Tabs active_index={active_index} onTabItemClick={setActiveTabIndex} top className='ssb'>
+                <div id='db-run-panel-tab__summary' label={localize('Summary')}>
+                    <Summary
+                        is_clear_dialog_visible={is_clear_dialog_visible}
```

Comment:
Since we are not using `is_clear_dialog_visible` in the summary and journal panel, can we remove it from the props? @maryia-matskevich-deriv

Reply:
sure bhai, removed✅